templates/keras_cv/index.md

```diff
@@ -43,9 +43,13 @@ preprocessing_model = keras.Sequential([
 Augment a `tf.data.Dataset`:
 
 ```python
-dataset = dataset.map(lambda images, labels: {"images": images, "labels": labels})
-dataset = dataset.map(preprocessing_model)
-dataset = dataset.map(lambda inputs: (inputs["images"], inputs["labels"]))
+def augment_data(images, labels):
+  inputs = {"images": images, "labels": labels}
+  for layer in preprocessing_layers:
+    inputs = layer(inputs)
+  return layer
```

Comment:
- `preprocessing_layers` as specified is a Sequential model, not a list of layers. Let's name it `preprocessing_stage`.
- We should return the last output, not the last layer
- The dataset should yield a tuple `(images, labels)` since we call `fit()` on it.
- Please try running the code in a Colab before submitting, to ensure some level of testing.

The code should be:

```python
dataset = dataset.map(lambda image, labels: (preprocessing_stage(images), labels))
...
densenet.fit(dataset)
```